src/chi-vue/src/components/drawer/drawer.tsx

```diff
@@ -107,7 +107,14 @@ export default class Drawer extends Vue {
     this.$emit(DRAWER_EVENTS.HIDE);
   }
 
-  _documentClickHandler = (ev: Event): void => {
+  @Watch('active')
+  activeChange(newValue: boolean, oldValue: boolean) {
+    if (!!newValue !== !!oldValue) {
```

Comment:
suggestion: if you don't expect these values not to exist I think it's not necessary to use the double exclamation:
```
if (newValue !== oldValue)
```